TimerCalculatorUtility/TimerCalculatorUtility/Views/TimerView.xaml

```diff
@@ -0,0 +1,105 @@
+            <converters:StartButtonVisibilityConverter x:Key="StartButtonVisibilityConverter"/>
+            <converters:LapButtonVisibilityConverter x:Key="LapButtonVisibilityConverter"/>
+            <converters:ResumetButtonVisibiliryConverter x:Key="ResumetButtonVisibiliryConverter"/>
+            <converters:ResetButtonVisibilityConverter x:Key="ResetButtonVisibilityConverter"/>
+            <converters:StopButtonVisibilityConverter x:Key="StopButtonVisibilityConverter"/>
+            <Color x:Key="PinkButton">#d4526a</Color>
+            <Color x:Key="BlueButton">#6063f0</Color>
+        </ResourceDictionary>
+    </ContentPage.Resources>
+    
+    <ContentPage.BindingContext>
+        <viewmodels:TimerViewModel x:Name="ViewModel"/>
+    </ContentPage.BindingContext>
+    
+    <ContentPage.Content>
+        <AbsoluteLayout>
+            <Label Text="{Binding TimeString}"
+                   FontSize="65"
+                   TextColor="{StaticResource Key=Black}"
+                   AbsoluteLayout.LayoutFlags="All"
+                   AbsoluteLayout.LayoutBounds="0.53,0.17,0.7,0.13"
+                   x:Name="CurrentTimeLabel"/>
+            <Label Text="{Binding LapTimeString}"
+                   FontSize="35"
+                   AbsoluteLayout.LayoutFlags="All"
+                   AbsoluteLayout.LayoutBounds="0.5, 0.3, 0.4, 0.08"/>
+         
+            <ListView ItemsSource="{Binding Times}"
+                      IsVisible="{Binding Path=Status, 
+                                                 Converter={StaticResource ListViewVisibilityConverter}}"
+                      AbsoluteLayout.LayoutFlags="All"
+                      AbsoluteLayout.LayoutBounds="0.5,0.7,0.8,0.5"                      
+                      >
+                <ListView.Header>
+                    <StackLayout Margin="0,30,0,20">
+                        <StackLayout Orientation="Horizontal" >
+                        <Label Text="Lap" HorizontalOptions="StartAndExpand"/>
+                        <Label Text="Lap times" HorizontalOptions="CenterAndExpand"/>
+                        <Label Text="Overall time" HorizontalOptions="EndAndExpand"/>
+                        </StackLayout>
+                        <BoxView HeightRequest="1" BackgroundColor="{StaticResource Key=White}"/>
+                    </StackLayout>
+                </ListView.Header>
+                <ListView.ItemTemplate>
+                    <DataTemplate>
+                        <ViewCell Height="100" >
+                            <StackLayout Orientation="Horizontal">
+                                <Label Text="{Binding LapNumber}" HorizontalOptions="StartAndExpand" />
+                                <Label Text="{Binding LapTime, StringFormat='{0:mm\\:ss\\.ff}'}" HorizontalOptions="CenterAndExpand" Margin="10,0,0,0"/>
+                                <Label Text="{Binding OverallTime, StringFormat='{0:mm\\:ss\\.ff}'}" HorizontalOptions="EndAndExpand" Margin="0,0,10,0" />
```

Comment:
The same as above. One property per line, right after another vertically, they should form a column